pkg/cmd/server/start.go

```diff
@@ -54,7 +56,11 @@ func (cfg Config) startMaster() error {
 		return err
 	}
 
-	glog.Infof("  Nodes: %v", cfg.NodeList)
+	glog.Infof("Nodes: %v", cfg.NodeList)
+
+	if strings.Contains(openshiftConfig.MasterAddr, "127.0.0.1") {
```

Comment:
Yes - the most common case is "no non-loop back ip", but it's not totally invalid.


> On Mar 13, 2015, at 7:43 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/cmd/server/start.go:
> 
> > @@ -54,7 +56,11 @@ func (cfg Config) startMaster() error {
> >  		return err
> >  	}
> >  
> > -	glog.Infof("  Nodes: %v", cfg.NodeList)
> > +	glog.Infof("Nodes: %v", cfg.NodeList)
> > +
> > +	if strings.Contains(openshiftConfig.MasterAddr, "127.0.0.1") {
> If the BindAddr includes other IPs, then a container that knew where it was located would be able to communicate, wouldn't it? We may not wire it up that way by default, but I'm pretty sure it could be done with a cleverly built .kubeconfig.
> 
> —
> Reply to this email directly or view it on GitHub.
>